api/v1/setup_teardown_routes_test.go

```diff
@@ -140,14 +142,14 @@ func TestSetupData(t *testing.T) {
 			runner.SetOptions(lib.Options{
 				Paused:          null.BoolFrom(true),
 				VUs:             null.IntFrom(2),
-				VUsMax:          null.IntFrom(2),
 				Iterations:      null.IntFrom(3),
+				NoSetup:         null.BoolFrom(true),
 				SetupTimeout:    types.NullDurationFrom(1 * time.Second),
 				TeardownTimeout: types.NullDurationFrom(1 * time.Second),
 			})
-			executor := local.New(runner)
-			executor.SetRunSetup(false)
```

Comment:
I don't understand why this was set before ... I think it should've never been set ?